src/OrderFormAcceptanceTests.Steps/Steps/CommonSteps.cs

```diff
@@ -40,18 +40,6 @@ public void ThenTheNewOrderIsPresented()
 
         [Given(@"mandatory data are missing")]
         [When(@"the User has not entered a Supplier search criterion")]
-        public void GivenMandatoryDataAreMissing()
-        {
-            //clear fields
-            //var listOfTextAreas = Test.Driver.FindElements(By.TagName("textarea"));
-            var listOfInputs = Test.Driver.FindElements(By.ClassName("nhsuk-input"));
-            foreach(var element in listOfInputs)
-            {
-                element.Clear();
-            }
-
-        }
-
```

Comment:
squiffy merge?
I was using that to clear values out of the forms

Reply:
yeah, I'll get it back in the commit